api/utils/constants.ts

```diff
@@ -3,3 +3,12 @@ export const AGENCIES_TABLE =
 
 export const CATEGORIES_TABLE =
   'handson_central_ohio__handson_central_ohio_agency_subcategory_terms';
+
+export const SCOS_HOST =
+  process.env.SCOS_HOST || 'https://data.smartcolumbusos.com';
+
+export const LOG_LEVEL = process.env.LOG_LEVEL || 'error';
+
+export const PORT = process.env.PORT || 8000;
+
+export const ENV = process.env.ENV || 'local';
```

Comment:
This is one place where we can set the constants that use `process.env` for anything. We shouldn't really be accessing the `process.env` outside of this file.